chapters/classes.tex

```diff
@@ -373,10 +373,10 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 parameter Boolean b = false;
 
 /* Variability analysis might cause evaluation of n: */
-parameter Real x = if b and n > 3 then x else 0;
+parameter Real x = if b and n > 3 then 1 - x else 0;
```

Comment:
With the latest revision would this example be allowed still?

Reply:
Again, GitHub seems to be doing strange things to your comments, so I'm quoting what I'm answering to:
> It is a good name. Is it defined already?

I think you are referring to _variability analysis_?  No, it's not defined as far as I know, it just comes natural to me.  Of course, we should't use undefined terms like this, so I suggest we explain it under 3.8 _Variability of Expressions_.

Reply:
Now the other comment:
> With the latest revision would this example be allowed still?

Good catch!  The concept driving parameter evaluation has changed from variability analysis to fulfillment of the acyclic binding rule.  Better now?